zrml/prediction-markets/src/lib.rs

```diff
@@ -938,6 +947,90 @@ mod pallet {
             let max_cats: u32 = T::MaxCategories::get().into();
             Self::calculate_actual_weight(T::WeightInfo::sell_complete_set, assets_len, max_cats)
         }
+
+        /// When the `MaxDisputes` amount of disputes is reached, this allows to start a global dispute.
+        ///
+        /// # Arguments
+        ///
+        /// * `market_id`: The identifier of the market.
+        ///
+        /// NOTE:
+        /// The outcomes of the disputes and the report outcome is added to the global dispute voting outcomes.
+        /// The bond of each dispute is the initial vote amount.
+        #[pallet::weight(T::WeightInfo::start_global_dispute(CacheSize::get()))]
+        #[transactional]
+        pub fn start_global_dispute(
+            origin: OriginFor<T>,
+            #[allow(dead_code, unused)]
+            #[pallet::compact]
+            market_id: MarketIdOf<T>,
+        ) -> DispatchResultWithPostInfo {
+            ensure_signed(origin)?;
+
+            #[allow(dead_code, unused)]
+            let mut weight: Weight = T::WeightInfo::start_global_dispute(CacheSize::get());
+
+            #[cfg(feature = "with-global-disputes")]
+            {
```

Comment:
Can you validate, that this works as intended?

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/682/commits/835e6fee074abf9859b20db87e5641b8dec51705